v20.1/inverted-indexes.md

```diff
@@ -4,14 +4,18 @@ summary: Inverted indexes improve your database's performance and usefulness by
 toc: true
 ---
 
-Inverted indexes improve your database's performance by helping SQL locate the schemaless data in a [`JSONB`](jsonb.html) column.
+An inverted index stores a mapping from a value to its location in a schemaless data structure.  Inverted indexes are commonly used in [document retrieval systems](https://en.wikipedia.org/wiki/Document_retrieval).
```

Comment:
I don't think this is exactly correct. An inverted index accelerates containment searches into columns that are container types. I'm struggling to phrase it. But the key thing is that it specifically works for containment searches.

If you want to stick to the mapping phrasing, you could say that it stores a mapping from a value within a container column to the row that has that container, or something. That's not great either though!

I do think that "schemaless" should be removed from this though - I don't think it's all that accurate.

Reply:
I tried rephrasing this intro part quite a bit using a simplified (?) version of your comment "it stores a mapping from a value within a container column ...".  PTAL.